gremlin-scala/src/main/scala/gremlin/scala/ScalaGraph.scala

```diff
@@ -46,12 +48,12 @@ case class ScalaGraph[G <: Graph](graph: G) {
   def +(label: String, properties: (String, Any)*): ScalaVertex = addVertex(label, properties.toMap)
 
   // get vertex by id
-  def v(id: AnyRef): Option[ScalaVertex] =
-    graph.traversal.V(id).headOption map ScalaVertex.apply
+  def v(id: Any): Option[ScalaVertex] =
+    graph.traversal.V(id.asInstanceOf[AnyRef]).headOption map ScalaVertex.apply
```

Comment:
I didn't know you can just cast e.g. an `Int` to `AnyRef`, but it does work apparently. Do you know if this is always safe?